test/mango.py

```diff
@@ -32,8 +33,7 @@ def __init__(self, host, port, dbname, auth=None):
         self.port = port
         self.dbname = dbname
         self.sess = requests.session()
-        if auth is not None:
-            self.sess.auth = auth
+        self.sess.auth = ('adm', 'pass')
```

Comment:
It uses admin party e.g. none. I'm fine with adm/pass, just would be nice to mention them somehow. However, who reads the docs today?(: